src/main/java/pl/ioad/adoto/database/dto/GeometryDTO.java

```diff
@@ -5,11 +5,4 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-@NoArgsConstructor
-@AllArgsConstructor
-@Getter
-@Setter
-public class GeometryDTO {
-    private Double[][] coordinates;
-    private String type;
-}
+public record GeometryDTO (double[][] coordinates, String type) { }
```

Comment:
It's only proposition, but maybe it will be easier to operate on `List<Pair<Double, Double>>` with streams instead of `dobule[][]` 😉